### DISCUSSION TITLE
Asking for help, NPM got an internal error while applying for an SSL certificate

### DISCUSSION THREAD

**Discussion by FrankleeM** [NginxProxyManager/nginx-proxy-manager#2472] [Q&A]
The error message is as follows:

Error: Command failed: certbot certonly --config "/etc/letsencrypt.ini" --cert-name "npm-1" --agree-tos --email "happylife767@163.com" --domains "*.frankcloud.vip,frankcloud.vip" --authenticator dns-dnspod --dns-dnspod-credentials "/etc/letsencrypt/credentials/credentials-1"
An unexpected error occurred:
ModuleNotFoundError: No module named 'zope'
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /tmp/certbot-log-1tjhr58o/log or re-run Certbot with -v for more details.

    at ChildProcess.exithandler (node:child_process:399:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)

![Snipaste_2022-12-12_15-49-10](https://user-images.githubusercontent.com/32917028/206991788-59a7e085-28e0-4147-8a3d-d7f768e56aea.png)

**Comment by FrankleeM** [comment]
Resolved
using image github-pr-2411

**Reply by passer77** [reply]
how?

**Reply by GitMeier** [reply]
https://github.com/NginxProxyManager/nginx-proxy-manager/pull/2411

Worked for me as well. Replaced the line
    image: 'jc21/nginx-proxy-manager:latest'
in my docker-compose.yaml file with
    image: 'jc21/nginx-proxy-manager:github-pr-2411'
